plugins/externalwrench/CMakeLists.txt

```
@@ -8,10 +8,16 @@ PROJECT(Plugin_ExternalWrench)
 
 include(AddGazeboYarpPluginTarget)
 
+set(LIB_COMMON_NAME gazebo_yarp_lib_common)
+if(CMAKE_VERSION VERSION_LESS 3.0.0)
+    get_property(GAZEBO_YARP_COMMON_HEADERS GLOBAL PROPERTY GAZEBO_YARP_COMMON_HEADERS)
+    unset(LIB_COMMON_NAME)
+endif()
+
 add_gazebo_yarp_plugin_target(LIBRARY_NAME externalwrench
                               INCLUDE_DIRS include/gazebo
                               SYSTEM_INCLUDE_DIRS ${Boost_INCLUDE_DIRS} ${GAZEBO_INCLUDE_DIRS}  ${SDFORMAT_INCLUDE_DIRS} ${PROTOBUF_INCLUDE_DIRS}
-                              LINKED_LIBRARIES ${YARP_LIBRARIES} ${GAZEBO_LIBRARIES} ${Boost_LIBRARIES}
+                              LINKED_LIBRARIES ${LIB_COMMON_NAME} ${YARP_LIBRARIES} ${GAZEBO_LIBRARIES} ${Boost_LIBRARIES}
```

I think just adding  `gazebo_yarp_lib_common` here is sufficient. CMake 3.5 is now the minimum required version, and I do not see the value added by `LIB_COMMON_NAME` value otherwise. 
```suggestion
                              LINKED_LIBRARIES gazebo_yarp_lib_common ${YARP_LIBRARIES} ${GAZEBO_LIBRARIES} ${Boost_LIBRARIES}
```

---

@traversaro Done https://github.com/robotology/gazebo-yarp-plugins/pull/422/commits/2a5dc3ca27a164f3353f8dd000306710d2ca2f96